pkg/pool-weighted/contracts/WeightedPool.sol

```diff
@@ -67,6 +67,7 @@ contract WeightedPool is BaseWeightedPool, InvariantGrowthProtocolFees {
         uint256[] memory normalizedWeights,
         address[] memory assetManagers,
         uint256 swapFeePercentage,
+        IProtocolFeePercentagesProvider protocolFeeProvider,
```

Comment:
I guess it doesn't matter, but I had this parameter right under the Vault, like in Stable

Reply:
Yep, I just noticed that ManagedPool does the same. I'll move this up to match.